app/common/components/resource/resource-list/resource-list-config.factory.js

```diff
@@ -46,9 +46,10 @@ function resourceListConfig ($filter, table) {
     constructor (resource) {
       const textPhysical = utils.Naming.humanize(resource.physical || '')
       const textTrading = utils.Naming.humanize(resource.trading || '')
+      const textUsage = utils.Naming.humanize(resource.usage || '')
       const textAllocated = resource.allocated ? utils.Naming.humanize('allocated') : null
      
-      const content = [textPhysical, textTrading, textAllocated].filter(a => a).join(' / ')
+      const content = [textPhysical, textTrading, textUsage, textAllocated].filter(a => a).join(' / ')
```

Comment:
if line 50 is removed (see comment above), then `textAllocated` in this line should also be removed